lib/server/cms-data.ts

```diff
@@ -68,3 +68,12 @@ export async function getGuideContent(
   const jsonData = await getGuideCmsData();
   return jsonData["guides"][section][name]?.content;
 }
+
+export async function getRosterCmsData(): Promise<any> {
```

Comment:
it would be helpful to have a real type as the return value here.

Reply:
Yeah... I can change this when I merge the two jsons together.